libs/dev_only_scripts/configToDb.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+require_once '/var/www/html/libs/php-commons/Db.php';
+
+$dir = '/var/www/html';
+
+checkTemplate($dir);
+
+function checkTemplate($folder) {
+    //if (is_dir($folder . '/.svn')) {
+    if (is_dir($folder . '/sources')) {

```

Comment:
This should be .svn otherwise this will take a very long time since it will go even deeper.